src/peft/tuners/lora/layer.py

```diff
@@ -239,6 +261,22 @@ def _apply_dora(self, x, lora_A, lora_B, scaling, active_adapter):
 
         return result_dora
 
+    def _cal_wlora_scale(self, active_adapter: str) -> torch.Tensor:
```

Comment:
Let's rename this to `_calculate_wlora_scale` or `_get_wlora_scale` (and replace "wlora" with whatever other name you choose).